src/components/HeaderNav/HeaderNav.module.css

```
@@ -48,6 +48,11 @@ ul.dropdownList {
   justify-content: space-between;
 }
 
+.publicWrapper {
+  padding: 0 2rem;
+  border-bottom: 2px solid transparent;
```

No px in our hood ;)
```suggestion
  border-bottom: 0.2rem solid transparent;
```